tests/lametro/test_bills.py

```diff
@@ -46,11 +56,17 @@ def test_private_scrape_dates(scraper, matter, intro_date, num_bills_scraped, mo
     matter['MatterIntroDate'] = intro_date
     matter['MatterRestrictViewViaWeb'] = True
     matter_id = matter['MatterFile']
-    mocker.patch('lametro.LametroBillScraper.matter', return_value=matter)
 
-    scrape_results = []
-    for bill in scraper.scrape(matter_ids=matter_id):
-        if type(bill) == Bill:
-            scrape_results.append(bill)
+    with requests_mock.Mocker() as m:
+        matcher = re.compile('webapi.legistar.com')
+        m.get(matcher, json={}, status_code=200)
```

Comment:
ok this is really cool!

Reply:
I KNOW. https://requests-mock.readthedocs.io/en/latest/matching.html#regular-expressions